stable_baselines3/common/off_policy_algorithm.py

```diff
@@ -431,7 +465,11 @@ def collect_rollouts(
                         reward_ = self._vec_normalize_env.get_original_reward()
                     else:
                         # Avoid changing the original ones
-                        self._last_original_obs, new_obs_, reward_ = self._last_obs, new_obs, reward
+                        self._last_original_obs, new_obs_, reward_ = (
```

Comment:
as mentioned in my comments in the main, please revert such formatting change (where a trailing comma was added) on code not related to your PR ;)

Reply:
I think this is issue with line lengths? Make sure you provide `-l 127` in arguments when running `black`.

Reply:
which version of black is that? do you specify the `-l 127` argument?
and to reformat, you need to remove the last trailing comma before calling black, otherwise, black will keep it as-is.

We are using `black==20.8b1`.

in the worst case, I will see how to checkout part of the file the way they are in master.

Reply:
I use black version `20.8b1` as well. I was forgetting to run black at all so I added a hook to VS code which runs it "on save". So since I wasn't running `black` explicitly, it was using the default 88. Before committing, I'd then run `black -l 127 .` but I am still potentially missing something as the mentioned comma issue resulted after doing this.

On that note though, is there consideration for using a `pyproject.toml` to have the black configuration explicit in the repo? (See [this comment](https://github.com/psf/black/issues/290#issuecomment-679841004))

Reply:
>On that note though, is there consideration for using a pyproject.toml to have the black configuration explicit in the repo? (See this comment)

not really as we have already a `setup.cfg` file and that the `Makefile` is normally in charge of handling those default (although it works only on linux/mac os and probably on Windows Subsystem for Linux ).
but we may consider it in the future.

If possible, I would highly recommend to run `make commit-checks` before committing (it runs every checks except the tests), but it may be trickier on windows...

Reply:
and what is your python version? (I'm using python 3.6 locally)

Reply:
> If possible, I would highly recommend to run make commit-checks before committing
Will try

I'm on python 3.8